Ahmadreza-tahmasebi/c5.js

```diff
@@ -0,0 +1,42 @@
+"use strict"
+
+
+function mergeSort(nums) {
+    if (nums.length === 0) {
+        return []
+    }
+    if (nums.length === 1)
+        return nums
+    var firstPart = mergeSort(nums.slice(0, nums.length / 2));
+    var secondPart = mergeSort(nums.slice(nums.length / 2));
+    var res = []
+    var firstHead = 0;
+    var secHead = 0;
+    while(1){
```

Comment:
Could you please explain why you have used `while(1)` here.

Reply:
run the following code until `break` is compiled.